aten/src/ATen/native/Distance.cpp

```diff
@@ -85,17 +85,17 @@ static Tensor cdist_impl(const Tensor& x1, const Tensor& x2, const double p, c10
   TORCH_CHECK(device1 == device2, "X1 and X2 must have the same device type. X1: ", device1, " X2: ", device2);
   // TODO: This is bad; this test should apply universally
   TORCH_CHECK(!x1.is_cuda() || x1.get_device() == x2.get_device(), "device of X1 (", x1.get_device(), ") must match device of X2 (", x2.get_device(), ")");
-  int64_t c1 = x1.size(-1);
-  int64_t c2 = x2.size(-1);
+  SymInt c1 = x1.sym_size(-1);
+  SymInt c2 = x2.sym_size(-1);
```

Comment:
`cdist_impl` (not in the yaml file -- cannot use dispatch) is called from `cdist` (`CompositeImplicitAutograd`). So cannot write Python metas for these, need to SymInt instead.